Makefile

```diff
@@ -15,5 +15,5 @@ clean: style
 
 # Test
 .PHONY: test
-test:
-	pytest -v --cov-config=pyproject.toml
```

Comment:
This line should not be commented.

Reply:
It didn't work on my setup with this config, but changed it back now. Thanks!

Reply:
Please remove this file from your changes too since backtracking errors would be hard. So, basically this file should not appear in your changes.

Reply:
Thanks! I removed the makefile from tracking. Hope this is what you meant?

Reply:
Hey, @ahoimarie, this file should not be shown as a part of your changes.